packages/jsActions/nanoflow-actions-native/src/external/DownloadFile.ts

```diff
@@ -0,0 +1,51 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+import fetchBlob from "rn-fetch-blob";
+
+// BEGIN EXTRA CODE
+function formatMendixFileUrl(file: mendix.lib.MxObject): string {
+    return `${mx.remoteUrl}file?guid=${file.getGuid()}&changedDate=${
+        file.get("changedDate") ?? ""
+    }&name=${encodeURIComponent(file.get("Name"))}`;
+}
+
+function formatPath(...pathArgs: string[]): string {
+    return pathArgs.filter(arg => !!arg).join("/");
+}
+
+function validateFileName(name: string) {
```

Comment:
I would go for `sanitizeFileName`, which conveys more clearly what's happening here. (Validation suggests that it might throw an error or something.)

Reply:
:sparkles: